Telemetry/Telemetry.swift

```diff
@@ -23,7 +23,8 @@ public class Telemetry {
         switch AppConstants.BuildChannel {
         case .Fennec: fallthrough
         case .Aurora: fallthrough
-        case .Developer: channel = "default"
+        case .Unknown: channel = "default"
```

Comment:
NBD, but I think it would have made more sense if I had created a `default: "default"` here instead of these fallthroughs. I mean, it's even called `default`!